src/components/cc-pricing-product/cc-pricing-product.js

```diff
@@ -111,9 +111,17 @@ export class CcPricingProduct extends withResizeObserver(LitElement) {
    * @return {string|undefined} the translated feature name if a translation exists or nothing if the translation does not exist
    */
   _getFeatureName (feature) {
-    if (feature != null && FEATURES_I18N[feature.code] != null) {
+    if (feature == null) {
+      return '';
+    }
+
+    if (FEATURES_I18N[feature.code] != null) {
       return FEATURES_I18N[feature.code]();
     }
+
+    if (feature.name != null) {
+      return feature.name;
+    }
```

Comment:
Maybe we should always try to return something instead of `void`: use `feature.code` as a fallback.

```suggestion
    return feature.name ?? feature.code;
```

Reply:
(see `cc-pricing-estimation` comment for reasons why I chose not to change this even though it was a good idea)